src/bioregistry/data/bioregistry.json

```diff
@@ -118970,6 +118970,7 @@
       "name": "Worm Base",
       "prefix": "426"
     },
+    "pattern": "^(CE[0-9]{5}|WB[A-Z][a-z]+\\d+)$",
```

Comment:
This is from a commit that has already been done on another branch so should be removed from this one